core/foundation/inc/ROOT/RLogger.hxx

```diff
@@ -16,28 +16,62 @@
 #ifndef ROOT7_RLogger
 #define ROOT7_RLogger
 
-#include <array>
+#include <atomic>
 #include <memory>
+#include <mutex>
 #include <sstream>
 #include <vector>
```

Comment:
```suggestion
#include <vector>
#include <string>
```

to be very pedantic something for `std::swap` should also be included (and to be even more pedantic something for `std::move` :D )